pkg/execution/executor/executor.go

```diff
@@ -2969,8 +2972,6 @@ func (e *executor) RetrieveAndScheduleBatch(ctx context.Context, fn inngest.Func
 
 	if md != nil {
 		span.SetAttributes(attribute.String(consts.OtelAttrSDKRunID, md.ID.RunID.String()))
-	} else {
-		span.SetAttributes(attribute.Bool(consts.OtelSysStepDelete, true))
```

Comment:
question, non-blocking: Do we still want to handle this or leave this blank and then if we don't see the delete/unset status we can investigate separately as another issue?

Reply:
it's been moved upwards to mark as delete on errors.
so if this still happens, we should definitely investigate.